package.json

```diff
@@ -79,38 +101,19 @@
     "webpack-config-single-spa-react-ts": "^4.0.1",
     "webpack-config-single-spa-ts": "^4.1.0",
     "webpack-dev-server": "^4.0.0-rc.0",
-    "webpack-merge": "^5.8.0"
+    "webpack-merge": "^5.8.0",
+    "yup": "^0.32.11"
   },
   "overrides": {
     "cql-execution": "2.3.3"
   },
   "dependencies": {
-    "@date-io/date-fns": "^2.13.1",
```

Comment:
just curious about the reasoning for moving all of these to devDependencies?

Reply:
I did some testing afterwards.  From what I've seen from other projects they should be de only anyways.. but I'll check more